queries/julia/indents.scm

```diff
@@ -48,3 +48,10 @@
   (block_comment)
 ] @indent.ignore
 
+((argument_list) @indent.align
+  (#set! indent.open_delimiter "(")
+  (#set! indent.close_delimiter ")"))
+
+((parameter_list) @indent.align
+  (#set! indent.open_delimiter "(")
+  (#set! indent.close_delimiter ")"))
```

Comment:
Could you add the expected behavior as a test case? It is sufficient to place your sample as a Júlia filé in the indent test folder for Julia (if it exists already for that language)

Reply:
Perfect! I will do this!

Reply:
Hi @theHamsta !

I think I add all the required tests :)